semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/DbIncubatingAttributes.java

```diff
@@ -398,16 +434,20 @@ public final class DbIncubatingAttributes {
    * <p>Notes:
    *
    * <p>For sanitization see <a
-   * href="../database/database-spans.md#sanitization-of-dbquerytext">Sanitization of {@code
+   * href="/docs/database/database-spans.md#sanitization-of-dbquerytext">Sanitization of {@code
```

Comment:
@open-telemetry/weaver-maintainers @lmolkova what do you think of having weaver automatically turn this into a fully-qualified link, e.g.

https://github.com/open-telemetry/semantic-conventions/blob/v1.34.0/docs/database/database-spans.md#sanitization-of-dbquerytext